Jenkinsfile

```diff
@@ -53,6 +53,22 @@ pipeline {
             make install
           '''
         }
+        dir ('proxygen') {
+          checkout([$class: 'GitSCM',
+          branches: [[name: '*/master']],
```

Comment:
I would probably look for a release tag since master will likely break our build at some point.